### PR TITLE
[DOCU-3188] OSS Client Portal 

### PR DESCRIPTION
**Pull request by cloudjumpercat**
### Description

What did you change and why?
- Rearranged some of the Dev Portal section to make a spot for the self-hosting pages
- Added the new "About Self Hosting" page
- Added the new "How to self host with Netlify" page
 
Include any supporting resources, e.g. link to a Jira ticket, GH issue, FTI, Slack, Aha, etc.
DOCU-3188

### Testing instructions

Netlify link: 

https://deploy-preview-5631--kongdocs.netlify.app/konnect/dev-portal/customization/
https://deploy-preview-5631--kongdocs.netlify.app/konnect/dev-portal/customization/self-hosted-portal/
https://deploy-preview-5631--kongdocs.netlify.app/konnect/dev-portal/customization/netlify/

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

**Comment by devonl-kong** [comment]
Small typo "Customm" under https://deploy-preview-5631--kongdocs.netlify.app/konnect/dev-portal/customization/self-hosted-portal/

File: app/konnect/dev-portal/customization/netlify.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+* [A {{site.konnect_saas}} account](/konnect/getting-started/access-account/)
+* [Yarn 1.22.x installed](https://classic.yarnpkg.com/lang/en/docs/install/#mac-stable)
+* A [Netlify account](https://www.netlify.com/)
+
+## Configure your self-hosted domains in {{site.konnect_short_name}}
+
+1. In {{site.konnect_short_name}}, click {% konnect_icon dev-portal %} **Dev Portal** > [**Settings**](https://cloud.konghq.com/portal/portal-settings) in the sidebar.
+1. Click the **Portal Domain** tab.
+1. Enter a name for your custom portal domain API in the **Custom Hosted Domain** field. For example, `api.mycompany.com`. 
+1. Enter a name for your custom portal client domain in the **Customm Self-Hosted UI Domain** field. For example, `client.mycompany.com`. <!-- what's the difference between the client and API domain?-->
```
Comment (devonl-kong) [inline app/konnect/dev-portal/customization/netlify.md:22]: Small Typo here as well "Customm"

File: app/konnect/dev-portal/customization/netlify.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+1. Enter your custom portal client domain (`client.mycompany.com`) in the **Custom domain or subdomain** field. 
+1. Click **Verify**.
+1. Click **Domains** in the sidebar and then click the domain you just added.
+1. In the Name servers pane, note the listed name servers. Then, [configure the name servers in your domain registrar](https://docs.netlify.com/domains-https/netlify-dns/delegate-to-netlify/).
+
+Once your name servers are propogated (this may take a few minutes), your Dev Portal will now be deployed by Netlify.
+
+## Customize the example open source Dev Portal 
+
+If you want to further customize the example self-hosted Dev Portal, you can...
```
Comment (devonl-kong) [inline app/konnect/dev-portal/customization/netlify.md:68]: This should be rephrased or continued.

**Comment by cloudjumpercat** [comment]
@devonl-kong Thanks for the review! I also have a few other questions if you have the time:

- What features does the example application come with configured?
- What's the difference between the client and API domain?
- After you configure the domains in {{site.konnect_short_name}} and add a CNAME record to your domain's DNS records that points the custom API domain to the default portal domain, does that just mean self-hosting is enabled? Or what happens after that step?
- When you create the netlify.toml file, the doc mentioned “This is necessary for a single page application." what does that mean?

File: app/konnect/dev-portal/customization/netlify.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+      force = false
+    ```
+    This is necessary for a single page application.
+1. Push changes to the `prod` branch.
+1. In Netlify on the Team overview dashboard, click **Add new site** and then select **Import an existing project**. 
+1. Click **GitHub** for your Git provider.
+1. Grant Netlify access to GitHub and then click **Configure Netlify on Github**.
+1. When prompted, install Netlify on your GitHub account with the example Dev Portal repository.
+1. In Netlify, select the Dev Portal repository you just forked.
+1. Select "prod" from the **Branch to deploy** drop-down menu.
```
Comment (Guaris) [inline app/konnect/dev-portal/customization/netlify.md:55]:
```suggestion
1. Select **prod** from the **Branch to deploy** drop-down menu.
```

File: app/konnect/dev-portal/customization/netlify.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+1. Click **Show advanced**. 
+1. In the Environment variables section, click **New variable**.
```
Comment (Guaris) [inline app/konnect/dev-portal/customization/netlify.md:57]:
```suggestion
1. In the **Environment variables** section, click **New variable**.
```
If it's a literal section otherwise lowercase.

File: app/konnect/dev-portal/customization/netlify.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+      to = "/index.html"
+      status = 200
+      force = false
+    ```
+    This is necessary for a single page application.
+1. Push changes to the `prod` branch.
+1. In Netlify on the Team overview dashboard, click **Add new site** and then select **Import an existing project**. 
+1. Click **GitHub** for your Git provider.
+1. Grant Netlify access to GitHub and then click **Configure Netlify on Github**.
+1. When prompted, install Netlify on your GitHub account with the example Dev Portal repository.
```
Comment (Guaris) [inline app/konnect/dev-portal/customization/netlify.md:53]: This step should merge into the step before it.

Comment (Guaris) [reply app/konnect/dev-portal/customization/netlify.md:53]: or it can be used as a natural breaking point for the long list of instructions.

File: app/konnect/dev-portal/customization/self-hosted-portal.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+* **Fully customizable:** Use the [example frontend Dev Portal application](https://github.com/Kong/konnect-portal) as a starting point and then customize Dev Portal for your needs using the Portal API. You can also integrate the API specs with workflows tailored to your organization's own processes.
+* **Hosting service choice:** When you self-host, you also get to choose which hosting service you use to deploy your Dev Portal. 
+* **Range of customization options:** With the self-hosted Dev Portal, you determine how much you want to customize. You can choose to use the [example application](https://github.com/Kong/konnect-portal) right out of the box or you can use the Portal API to have more fine-grained control.
+
+## How the self-hosted Dev Portal works 
+
+The following diagram explains how the self-hosted Dev Portal works by using the example frontend application:
+
+![DIAGRAM HERE](/assets/images/docs/konnect/diagram.png)
```
Comment (Guaris) [inline app/konnect/dev-portal/customization/self-hosted-portal.md:23]: Diagram doesn't work
image

File: app/konnect/dev-portal/customization/self-hosted-portal.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+## Self-hosted Dev Portal benefits
+
+There are several benefits to keep in mind when deciding whether to use a {{site.konnect_short_name}}-hosted or self-hosted Dev Portal. The self-hosted portal provides the following benefits: 
+
+* **Fully customizable:** Use the [example frontend Dev Portal application](https://github.com/Kong/konnect-portal) as a starting point and then customize Dev Portal for your needs using the Portal API. You can also integrate the API specs with workflows tailored to your organization's own processes.
+* **Hosting service choice:** When you self-host, you also get to choose which hosting service you use to deploy your Dev Portal. 
+* **Range of customization options:** With the self-hosted Dev Portal, you determine how much you want to customize. You can choose to use the [example application](https://github.com/Kong/konnect-portal) right out of the box or you can use the Portal API to have more fine-grained control.
```
Comment (Guaris) [inline app/konnect/dev-portal/customization/self-hosted-portal.md:17]:
```suggestion
* **Range of customization options:** With the self-hosted Dev Portal, you determine how much you want to customize. You can choose to use the example application right out of the box or you can use the Portal API to have more fine-grained control.
```
Feels odd to link this a second time with a different name because as the reader I felt like I was going to be taken to a different link -- I already had the repo tab open from the first bullet's link. You can either remove the link, or keep the link but call it "example front end dev portal application" like you did in the first bullet.

**Review by Guaris — CHANGES_REQUESTED** [review]
https://deploy-preview-5631--kongdocs.netlify.app/konnect/dev-portal/customization/netlify/

The instructions in this doc need to be broken up at natural points and maybe even use H3's to introduce what's happening. 

Natural points I found were: 
step 8 could be a cut off point for github instructions
Step 9 starts netlify configuration instructions
Step 14 cuts off deplyoment
step 15 kicks off verification? 
Though it feels like steps 18-19 could be a sentence instead of a step. 


You can / should also merge some of those instructions. 

Steps 11-12 can be one step. 
steps 13-14 can be one step
step 16-17 can be one step 
potentially 18-19

File: app/konnect/dev-portal/customization/netlify.md [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+* [Yarn 1.22.x installed](https://classic.yarnpkg.com/lang/en/docs/install/#mac-stable)
+* [A Netlify account](https://www.netlify.com/)
+
+## Configure your self-hosted domains in {{site.konnect_short_name}}
+
+1. In {{site.konnect_short_name}}, click {% konnect_icon dev-portal %} **Dev Portal** > [**Settings**](https://cloud.konghq.com/portal/portal-settings) in the sidebar.
+1. Click the **Portal Domain** tab.
+1. Enter a name for your custom portal domain API in the **Custom Hosted Domain** field. For example, `api.mycompany.com`. 
+1. Enter a name for your custom portal client domain in the **Custom Self-Hosted UI Domain** field. For example, `client.mycompany.com`.
+1. After you configure these domains in {{site.konnect_short_name}}, add a CNAME record to your domain's DNS records that points the custom API domain to the default portal domain. For example:
```
Comment (lena-larionova) [inline app/konnect/dev-portal/customization/netlify.md:26]:
```suggestion
1. Add a CNAME record to your domain's DNS records that points the custom API domain to the default portal domain. For example:
```
By virtue of having numbered steps, the "after" is implied.

File: app/konnect/dev-portal/customization/netlify.md [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+    api.mycompany.com   CNAME   3600    client.mycompany.com
+    ```
+
+Now the self-hosted Dev Portal is enabled and the {{site.konnect_short_name}}-hosted Dev Portal is disabled. The custom portal domain API you specified will be used as the API domain of your portal and the custom portal client domain will be used to access the UI of your self-hosted portal.
+
+## Deploy your self-hosted Dev Portal with Netlify
+
+In this example, you will deploy the self-hosted example Dev Portal using Netlify. You can use any static site generator of your choice to host your Dev Portal.
+
+1. In GitHub, [fork a copy](https://docs.github.com/get-started/quickstart/fork-a-repo) of the [open source Dev Portal application example](https://github.com/Kong/konnect-portal).
```
Comment (lena-larionova) [inline app/konnect/dev-portal/customization/netlify.md:37]:
```suggestion
1. In GitHub, [fork](https://docs.github.com/get-started/quickstart/fork-a-repo) the [open source Dev Portal application example](https://github.com/Kong/konnect-portal).
```
Small nitpick - not forking a copy, you're forking the repo, as the fork is the copy.

File: app/konnect/dev-portal/customization/netlify.md [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+1. In the forked repository, [create a new branch](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-and-deleting-branches-within-your-repository#creating-a-branch) that you will use to deploy your Dev Portal. For example, `prod`. 
+    Upstream changes from `kong/konnect-portal` are merged into `main` and then finally merged into `prod` to trigger a deploy.
+1. On the `prod` branch, create a new `netlify.toml` file:
+    ```sh
```
Comment (lena-larionova) [inline app/konnect/dev-portal/customization/netlify.md:41]: I don't think `sh` is the right thing here. Is there support for `toml`? If not, I'd leave it blank.

File: app/konnect/dev-portal/customization/netlify.md [hunk 11]
```diff
@@ -0,0 +1,74 @@
...
+    [[redirects]]
+      from = "/*"
+      to = "/index.html"
+      status = 200
+      force = false
+    ```
+    This is necessary for a single page application.
+1. Push changes to the `prod` branch.
+1. In Netlify on the Team overview dashboard, click **Add new site** and then select **Import an existing project**.
```
Comment (lena-larionova) [inline app/konnect/dev-portal/customization/netlify.md:50]:
```suggestion
1. In Netlify, on the Team Overview dashboard, click **Add new site**, then select **Import an existing project**. 
```

File: app/konnect/dev-portal/customization/netlify.md [hunk 12]
```diff
@@ -0,0 +1,74 @@
...
+      from = "/*"
+      to = "/index.html"
+      status = 200
+      force = false
+    ```
+    This is necessary for a single page application.
+1. Push changes to the `prod` branch.
+1. In Netlify on the Team overview dashboard, click **Add new site** and then select **Import an existing project**. 
+1. Click **GitHub** for your Git provider.
+1. Grant Netlify access to GitHub and then click **Configure Netlify on Github**.
```
Comment (lena-larionova) [inline app/konnect/dev-portal/customization/netlify.md:52]:
```suggestion
1. Grant Netlify access to GitHub, then click **Configure Netlify on Github**.
```

File: app/konnect/dev-portal/customization/netlify.md [hunk 13]
```diff
@@ -0,0 +1,74 @@
...
+1. When prompted, install Netlify on your GitHub account with the example Dev Portal repository.
+1. In Netlify, select the Dev Portal repository you just forked.
+1. Select "prod" from the **Branch to deploy** drop-down menu.
+1. Click **Show advanced**. 
+1. In the Environment variables section, click **New variable**.
+1. Enter `VITE_PORTAL_API_URL` in the **Key** field and your custom portal domain API (`api.mycompany.com`) in the **Value** field.
+1. Click **Deploy site**. 
+1. Click **Domain settings** and **Add a domain**.
```
Comment (lena-larionova) [inline app/konnect/dev-portal/customization/netlify.md:60]:
```suggestion
1. Click **Domain settings**, then **Add a domain**.
```

File: app/konnect/dev-portal/customization/netlify.md [hunk 14]
```diff
@@ -0,0 +1,74 @@
...
+1. Enter your custom portal client domain (`client.mycompany.com`) in the **Custom domain or subdomain** field. 
+1. Click **Verify**.
+1. Click **Domains** in the sidebar and then click the domain you just added.
+1. In the Name servers pane, note the listed name servers. Then, [configure the name servers in your domain registrar](https://docs.netlify.com/domains-https/netlify-dns/delegate-to-netlify/).
```
Comment (lena-larionova) [inline app/konnect/dev-portal/customization/netlify.md:64]:
```suggestion
1. In the **Name servers** pane, note the listed name servers. Then, [configure the name servers in your domain registrar](https://docs.netlify.com/domains-https/netlify-dns/delegate-to-netlify/).
```

File: app/konnect/dev-portal/customization/netlify.md [hunk 15]
```diff
@@ -0,0 +1,74 @@
...
+
+Once your name servers are propogated (this may take a few minutes), your Dev Portal will now be deployed by Netlify.
```
Comment (lena-larionova) [inline app/konnect/dev-portal/customization/netlify.md:66]:
```suggestion
Once your name servers are propogated (this may take a few minutes), your Dev Portal will be deployed by Netlify.
```

File: app/konnect/dev-portal/customization/netlify.md [hunk 16]
```diff
@@ -0,0 +1,74 @@
...
+
+## Customize the example open source Dev Portal 
+
+If you want to further customize the example self-hosted Dev Portal, you can edit the settings in your forked copy of the open source {{site.konnect_short_name}} Dev Portal repository. For more information, see the [{{site.konnect_short_name}} Portal repository](https://github.com/Kong/konnect-portal).  
+
+## More information
+
+[About Self-Hosted Dev Portal](/konnect/dev-portal/customization/self-hosted-portal/) - Learn more about how the self-hosted Dev Portal works and why you should use it.
```
Comment (lena-larionova) [inline app/konnect/dev-portal/customization/netlify.md:74]:
```suggestion
* [About Self-Hosted Dev Portal](/konnect/dev-portal/customization/self-hosted-portal/) - Learn more about how the self-hosted Dev Portal works and why you should use it.
```

File: app/konnect/dev-portal/customization/self-hosted-portal.md [hunk 17]
```diff
@@ -0,0 +1,37 @@
+---
+title: About Self-Hosted Dev Portal
+content_type: explanation
+badge: oss
+---
+
+In {{site.konnect_saas}}, you have two hosting options for your Dev Portal: a cloud hosted Dev Portal with {{site.konnect_short_name}} or a self-hosted, open source Dev Portal. With the cloud hosted Dev Portal in {{site.konnect_short_name}}, your Dev Portal is hosted for you and it simplifies your deployment experience. The self-hosted, open source Dev Portal provides all the same features as the cloud-hosted portal, it just also gives you control over your hosting service and where your Dev Portal is hosted.
```
Comment (lena-larionova) [inline app/konnect/dev-portal/customization/self-hosted-portal.md:7]:
```suggestion
In {{site.konnect_saas}}, you have two hosting options for your Dev Portal: a cloud hosted Dev Portal with {{site.konnect_short_name}} or a self-hosted, open source Dev Portal. 
* **Cloud hosted Dev Portal in {{site.konnect_short_name}}**: Your Dev Portal is hosted for you and it simplifies your deployment experience. 
* **Self-hosted, open source Dev Portal**: Provides all the same features as the cloud-hosted portal, but also gives you control over your hosting service and where your Dev Portal is hosted. 
```
Splitting this paragraph into a list to make it easier to skim.

File: app/konnect/dev-portal/customization/self-hosted-portal.md (hunk 7 above)
Comment (lena-larionova) [inline app/konnect/dev-portal/customization/self-hosted-portal.md:17]:
```suggestion
* **Range of customization options:** With the self-hosted Dev Portal, you determine how much you want to customize. You can choose to use the [example application](https://github.com/Kong/konnect-portal) right out of the box, or use the Portal API for more fine-grained control.
```

File: app/konnect/dev-portal/customization/self-hosted-portal.md [hunk 18]
```diff
@@ -0,0 +1,37 @@
...
+The following diagram explains how the self-hosted Dev Portal works by using the example frontend application:
+
+![DIAGRAM HERE](/assets/images/docs/konnect/diagram.png)
+
+> Figure 1: Diagram that shows how the example frontend application communicates with the developer's browser and {{site.konnect_short_name}}. The self-hosted Dev Portal is hosted in your infrastructure and the application is served to the developer browser. The developer browser communicates with the Portal API for API requests and {{site.konnect_short_name}} supplies the corresponding content. 
+
+ 
+## Enable a self-hosted Dev Portal
+
+To enable the self-hosted Dev Portal in {{site.konnect_short_name}}, click {% konnect_icon dev-portal %} **Dev Portal** > [**Settings**](https://cloud.konghq.com/portal/portal-settings) in the sidebar and then click the **Portal Domain** tab. Once you specify a custom portal domain API in the **Custom Hosted Domain** field and a custom portal client domain in the **Custom Self-Hosted UI Domain** field, your self-hosted portal is enabled. After enabling the portal, you must configure the settings in your static site generator to deploy your Dev Portal.
```
Comment (lena-larionova) [inline app/konnect/dev-portal/customization/self-hosted-portal.md:30]:
```suggestion
You can enable the self-hosted Dev Portal through {% konnect_icon dev-portal %} **Dev Portal** > [**Settings**](https://cloud.konghq.com/portal/portal-settings), then setting up a custom domain in the **Portal Domain** tab. 

The following fields are required:
* Custom Hosted Domain
* Custom Self-Hosted UI Domain

After enabling the portal, you must configure the settings in your static site generator to deploy your Dev Portal.
```
Hard to follow the instructions in a paragraph.

File: app/konnect/updates.md [hunk 19]
```diff
@@ -9,6 +9,11 @@ an application that lets you manage configuration for multiple runtimes
 from a single, cloud-based control plane, and provides a catalog of all deployed
 services. [Try it today!](https://cloud.konghq.com/quick-start)
 
+## June 2023
+
+**Self-hosted, open source Dev Portal**
+: You can now self-host an open source Dev Portal on the static site of your choice. Kong provides an [example application](https://github.com/Kong/konnect-portal) you can use for an out-of-the-box experience with a self-hosted Dev Portal. You can also customize the self-hosted portal using the Portal Management and Portal APIs as well as the Portal SDK. For more information, see [About Self-Hosted Dev Portal](/konnect/dev-portal/customization/self-hosted-portal/).
```
Comment (lena-larionova) [inline app/konnect/updates.md:15]:
```suggestion
: You can now self-host an open source Dev Portal on the static site of your choice. Kong provides an [example application](https://github.com/Kong/konnect-portal) you can use for an out-of-the-box experience with a self-hosted Dev Portal. You can also customize the self-hosted portal using the Portal Management and Portal APIs, and the Portal SDK. For more information, see [About Self-Hosted Dev Portal](/konnect/dev-portal/customization/self-hosted-portal/).
```

**Review by lena-larionova — COMMENTED** [review]
Looks good, just missing a diagram on the `self-hosted-portal` page.

**Review by Guaris — CHANGES_REQUESTED** [review]

File: app/konnect/dev-portal/customization/self-hosted-portal.md [hunk 20]
```diff
@@ -0,0 +1,49 @@
...
+
+![DIAGRAM HERE](/assets/images/docs/konnect/diagram.png)
+
+> Figure 1: Diagram that shows how the example frontend application communicates with the developer's browser and {{site.konnect_short_name}}. The self-hosted Dev Portal is hosted in your infrastructure and the application is served to the developer browser. The developer browser communicates with the Portal API for API requests and {{site.konnect_short_name}} supplies the corresponding content. 
+
+The Dev Portal API is always hosted by {{site.konnect_short_name}}, no matter if you use the self-hosted or {{site.konnect_short_name}}-hosted Dev Portal. The Dev Portal API maps to the Portal Domain (and optional Custom Hosted Domain, which is a CNAME). Depending on which domain settings you configure and if you are self-hosting or not, the Dev Portal UI domain will differ.
+
+![DIAGRAM HERE](/assets/images/docs/konnect/diagram.png)
+
+> Figure 2: Diagram that shows that self-hosted and {{site.konnect_short_name}}-hosted Dev Portals use different UI domains, but both use the same {{site.konnect_short_name}}-hosted Dev Portal API.
```
Comment (cloudjumpercat) [inline app/konnect/dev-portal/customization/self-hosted-portal.md:32]: @Guaris If you have a chance, could you review this? I added lines 28-32 to accommodate a diagram Ross wanted to add. I hope that it makes sense even without the diagram added. Hopefully marketing wraps up those diagrams soon.

Comment (cloudjumpercat) [reply app/konnect/dev-portal/customization/self-hosted-portal.md:32]: Oh, and if that looks good (I also read through the whole thing again), could you approve this PR so I can hand it over to Ross?

**Review by Guaris — APPROVED** [review]

File: app/_assets/images/docs/konnect/konnect-dev-portal-urls.png
Comment (rosskukulinski) [inline app/_assets/images/docs/konnect/konnect-dev-portal-urls.png:1]: @cloudjumpercat I think this needs minor adjustment.

SaaS-UI --> Custom Hosted Domain should be Custom Hosted Domain (CNAME)
Self-Managed-UI --> the domain name should be `https://portal.mycompany.dev`

File: app/konnect/dev-portal/customization/netlify.md [hunk 21]
```diff
@@ -0,0 +1,83 @@
...
+
+## Configure your self-hosted domains in {{site.konnect_short_name}}
+
+1. In {{site.konnect_short_name}}, click {% konnect_icon dev-portal %} **Dev Portal** > [**Settings**](https://cloud.konghq.com/portal/portal-settings) in the sidebar.
+1. Click the **Portal Domain** tab.
+1. Enter a name for your custom portal domain API in the **Custom Hosted Domain** field. For example, `api.mycompany.com`. 
+1. Enter a name for your custom portal client domain in the **Custom Self-Hosted UI Domain** field. For example, `client.mycompany.com`.
+1. Add a CNAME record to your domain's DNS records that points the custom API domain to the default portal domain. For example:
+    ```sh
+    api.mycompany.com   CNAME   3600    client.mycompany.com
```
Comment (rosskukulinski) [inline app/konnect/dev-portal/customization/netlify.md:29]: These are wrong.  The CNAME needs to point the Custom Hosted Domain to the Konnect Portal Domain.  

more like 

```
api.mycompany.com   CNAME   3600    portal29asdfj278.us.portal.konghq.com
```

**Review by rosskukulinski — CHANGES_REQUESTED** [review]
Approved with minor adjustments.

File: app/konnect/dev-portal/customization/self-hosted-portal.md [hunk 22]
```diff
@@ -0,0 +1,39 @@
+---
+title: About Self-Hosted Dev Portal
+content_type: explanation
+badge: oss
```
[the remark this answers is not in the document]
Comment (cloudjumpercat) [reply app/konnect/dev-portal/customization/self-hosted-portal.md]: @rosskukulinski So the OSS portal client = example application (konnect-portal repo)? Can you just use the OSS portal client and enable the settings in the Konnect UI and do everything with open source (as long as you don't plan to use the APIs)?

[the remark this answers is not in the document]
Comment (rosskukulinski) [reply app/konnect/dev-portal/customization/self-hosted-portal.md]: I think we should just remove the badge entirely.  We changed course and are just making the Self-hosted dev portal available to all Konnect tiers.

But the self-hosted dev portal, while OSS, _will not do anything_ unless it's tied to a Konnect organization. Hence I don't think the OSS badge makes a ton of sense?

[the remark this answers is not in the document]
Comment (cloudjumpercat) [reply app/konnect/dev-portal/customization/self-hosted-portal.md]: Sounds good, I'll just remove the badge entirely!

File: app/konnect/dev-portal/customization/index.md [hunk 23]
```diff
@@ -20,6 +26,16 @@ From the **Appearance** menu you have the ability to modify the following option
 
 For details on the requirements for each customizable option, hover over the information (`i`) icon next to each item in the UI.
 
+### Complete customization using the open source Dev Portal
```
Comment (cloudjumpercat) [inline app/konnect/dev-portal/customization/index.md:29]: @rosskukulinski I added this section that contains info about how to configure the open source Dev Portal and some of the benefits. Was this what you had in mind?

File: app/konnect/dev-portal/customization/index.md [hunk 24]
```diff
@@ -6,6 +6,12 @@ content_type: how-to
 
 ## Appearance
 
+To customize Dev Portal, you have two options:
+* Basic customization using the Dev Portal [Appearance settings](#appearance) in the UI
+* Complete customization using the [open source Dev Portal client](/konnect/dev-portal/customization/self-hosted-portal/)
```
Comment (cloudjumpercat) [inline app/konnect/dev-portal/customization/index.md:11]: @rosskukulinski I also added these per your feedback, do these look okay to you?

Comment (rosskukulinski) [reply app/konnect/dev-portal/customization/index.md:11]: Perfect! Thanks

File: app/konnect/dev-portal/customization/index.md [hunk 25]
```diff
@@ -20,6 +26,16 @@ From the **Appearance** menu you have the ability to modify the following option
 
 For details on the requirements for each customizable option, hover over the information (`i`) icon next to each item in the UI.
 
+### Complete customization using the open source Dev Portal
+
+You can completely customize the Dev Portal using the [open source Dev Portal client](/konnect/dev-portal/customization/self-hosted-portal/). To enable the self-hosted Dev Portal, navigate to {% konnect_icon dev-portal %} **Dev Portal** > [**Settings**](https://cloud.konghq.com/portal/portal-settings), then set up a custom domain in the **Portal Domain** tab.
+
+This self-hosted portal provides the following benefits: 
+
+* **Fully customizable:** Use the [example frontend Dev Portal application](https://github.com/Kong/konnect-portal) as a starting point and then customize Dev Portal for your needs using the [Portal API](https://developer.konghq.com/spec/2aad2bcb-8d82-43b3-abdd-1d5e6e84dbd6/b4539157-4ced-4df5-affa-7d790baee356). You can also integrate the API specs with workflows tailored to your organization's own processes.
```
Comment (rosskukulinski) [inline app/konnect/dev-portal/customization/index.md:35]: Suggestion:

using the [Portal API](https://developer.konghq.com/spec/2aad2bcb-8d82-43b3-abdd-1d5e6e84dbd6/b4539157-4ced-4df5-affa-7d790baee356) and [Portal SDK](https://www.npmjs.com/package/@kong/sdk-portal-js)

File: app/konnect/dev-portal/customization/index.md [hunk 26]
```diff
@@ -29,6 +45,8 @@ To add a custom URL to Dev Portal, you need:
 * A domain and access to configure the domain's DNS `CNAME` records.
 * Your organization's auto-generated default Dev Portal URL.
 
+You can also choose to [self-host the Dev Portal with Netlify](/konnect/dev-portal/customization/netlify/).
```
Comment (rosskukulinski) [inline app/konnect/dev-portal/customization/index.md:48]: You can also choose to [self-host the Dev Portal with Netlify](/konnect/dev-portal/customization/netlify/) or any other static hosting service that supports single page applications.

File: app/konnect/dev-portal/customization/index.md (hunk 23 above)
Comment (rosskukulinski) [reply app/konnect/dev-portal/customization/index.md:29]: Great! Few small tweaks, but looking good.

**Comment by rosskukulinski** [comment]
@acgoldsmith can you or someone from your team help with merge conflicts on updates.md?

File: app/konnect/dev-portal/customization/netlify.md [hunk 27]
```diff
@@ -0,0 +1,72 @@
+---
+title: Self-Host your Dev Portal with Netlify
+content_type: how-to
+badge: oss
+---
+
+You can use the self-hosted, open source Dev Portal to display your APIs to developers on a self-hosted website. This guide shows you how to configure an example open source Dev Portal, which you can further customize by adjusting the code of the sample app and self-hosting it on a frontend application of your choice.
```
Comment (acgoldsmith) [inline app/konnect/dev-portal/customization/netlify.md]:
```suggestion
You can use the self-hosted, open source Dev Portal to offer a fully-customizable user experience for your developers. This guide shows you how to configure an example open source Dev Portal, which you can further customize by adjusting the code of the sample app and self-hosting it on a frontend application of your choice.
```

File: app/konnect/dev-portal/customization/self-hosted-portal.md [hunk 28]
```diff
@@ -0,0 +1,39 @@
...
+---
+
+In {{site.konnect_saas}}, you have two hosting options for your Dev Portal: a cloud hosted Dev Portal with {{site.konnect_short_name}} or a self-hosted, open source Dev Portal. With the cloud hosted Dev Portal in {{site.konnect_short_name}}, your Dev Portal is hosted for you and it simplifies your deployment experience. The self-hosted, open source Dev Portal provides all the same features as the cloud-hosted portal, it just also gives you control over your hosting service and where your Dev Portal is hosted.
```
Comment (acgoldsmith) [inline app/konnect/dev-portal/customization/self-hosted-portal.md]:
```suggestion
In {{site.konnect_saas}}, you have two out-of-the box solutions for your Dev Portal: a cloud-hosted Dev Portal with {{site.konnect_short_name}} or a self-hosted, open source Dev Portal. With the cloud-hosted Dev Portal in {{site.konnect_short_name}}, your Dev Portal is hosted for you and it simplifies your deployment experience. The self-hosted, open source Dev Portal provides all the same features as the cloud-hosted portal, it just also gives you control over your hosting service and where your Dev Portal is hosted. 
```

**Review by acgoldsmith — COMMENTED** [review]
incorporating final feedback

**Review by acgoldsmith — APPROVED** [review]
LGTM on final updates
